src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -119,6 +117,17 @@ public override int Run(TOptions options)
                     ExecutionException = ex;
                     return FAILURE;
                 }
+
+                try
+                {
+                    PostLogFile(_rootContext, options, FileSystem);
```

Comment:
What is our in memory option for posting log file for the scenarios were we do not or cannot write to disk.

For example, for scanning in memory to call AnalyzeCommand.AnalyzeTargetHelper() and to dynamically validate we just result.Visit. 

Can we add a AnalyzeCommand helper method to post a SarifLog as in memory json string to the endpoint.